apps/src/code-studio/initApp/project.js

```diff
@@ -634,6 +634,8 @@ var projects = module.exports = {
       case 'gamelab':
         if (appOptions.droplet) {
           return 'gamelab';
+        } else if (appOptions.level.isDanceLab) {
+          return 'dancelab';
```

Comment:
I believe the standalone app is at `/p/dance`?

Reply:
Yes, it is!  Thanks